.circleci/config.yml

```diff
@@ -760,6 +760,38 @@ jobs:
             echo "== Javascript =="
             ls ./build/js/*.js
 
+  cookiecutter-bootstrap:
+    docker:
+      - image: cimg/python:3.10-node
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: ~/project
+    steps:
+      - run:
+          name: Install cookiecutter
+          command: pip install cookiecutter
+      - run:
+          name: Create a project from richie cookiecutter template
+          command: cookiecutter gh:openfun/richie --directory cookiecutter --no-input organization=fun
```

Comment:
you have to run cookiecutter on the template in the source code and not from github otherwise you get the one on master and not the one you are trying to merge :man_shrugging: